app/views/analytics/districts/show.csv.erb

```diff
@@ -88,6 +89,7 @@ end).to_csv -%>
     <%=
```

Comment:
![image](https://user-images.githubusercontent.com/16774200/83999944-63e44100-a981-11ea-9e33-4427de64c3d8.png)

The `All` row is missing a `nil` value under facility type.